src/Bridge/Symfony/Bundle/Resources/config/swagger.xml

```diff
@@ -30,7 +30,7 @@
         <!-- Action -->
 
         <service id="api_platform.swagger.action.ui" class="ApiPlatform\Core\Bridge\Symfony\Bundle\Action\SwaggerUiAction">
-            <argument type="service" id="twig" on-invalid="ignore" />
+            <argument type="service" id="twig" />
```

Comment:
What we should do for now is **not** registering the `SwaggerUiListener` if Twig isn't installed.